examples/js/archive.js

```diff
@@ -8,6 +9,14 @@ const input = document.getElementById('input');
 const responseText = document.getElementById('response');
 const imageContainer = document.getElementById('imgContainer');
 const mapToggle = document.getElementById('mapToggle');
+const prevBtn = document.getElementById('prevBtn');
```

Comment:
These might be better defined within pagination.js, what do you think? Just keep everything related to pagination hidden in there? (i.e. does the overall code need to know about these?)

Reply:
Alright @jywarren